### PR TITLE
Issue #63, patch for part of unix 

### PR DESCRIPTION
**Pull request by evileye2000**
#63

1. Add parseInitializeWithServerURL() to include/parse.h
2. Add parseInitializeWithServerURL() implementation to unix/src/parse

### CODE REVIEW

**Comment by evileye2000** [comment]
@nlutsenko @francip  please review this patch about issue #63

File: unix/src/parse.c [hunk 1]
```diff
@@ -539,7 +585,13 @@ static void parseSendRequestInternal(
         }
     }
 
-    int urlSize = strlen("https://api.parse.com");
+    int urlSize = 0;
+    if(clientInternal->serverURL != NULL){
```
Comment (francip) [inline unix/src/parse.c]: If you refactor the initialize method, this change becomes trivial and you don't need the if

File: unix/src/parse.c [hunk 2]
```diff
@@ -118,6 +120,50 @@ static size_t curlDataCallback(void *contents, size_t size, size_t nmemb, void *
...
+        client->lastPushTime = strdup(temp);
+    }
+
+    curl_global_init(CURL_GLOBAL_ALL);
+
+    return (ParseClient)client;
+}
+
 ParseClient parseInitialize(
         const char *applicationId,
```
Comment (francip) [inline unix/src/parse.c]: Refactor this method to call the one above with the default server URL. This way the initialization logic will not be duplicated.

**Comment by evileye2000** [comment]
I refactor initialize method and request logic that you said.

**Comment by evileye2000** [comment]
@nlutsenko @francip  would you please review and accept about this PR

**Comment by nlutsenko** [comment]
Deferring to @francip.

File: unix/src/parse.c [hunk 3]
```diff
@@ -551,7 +566,15 @@ static void parseSendRequestInternal(
         if (callback != NULL) callback(client, ENOMEM, 0, NULL);
         return;
     }
-    strncat(fullUrl, "https://api.parse.com", urlSize);
+    
+    if(clientInternal->serverURL != NULL){
+        strncat(fullUrl, clientInternal->serverURL, urlSize);
+    }
+    else {
```
Comment (francip) [inline unix/src/parse.c]: [nitpick] With the refactor of the initialize this branch should not be needed.

**Comment by francip** [comment]
LGTM

**Pull request merged**